### PR TITLE
formValues: add decorator

### PR DESCRIPTION
**Pull request by wmertens**
You can either use `formValues(path1, path2, ...)` or `formValues({prop1: path1, prop2: path2, ...})` to get values from the current form as props.

Implements #2996

### CODE REVIEW

**Comment by wmertens** [comment]
Updated with more tests.

**Comment by erikras** [comment]
Looks good. Can you write the docs for this?

File: src/immutable/formValues.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import createValues from '../createValues'
```
Comment (erikras) [inline src/immutable/formValues.js]: Should this not be `'../createFormValues'`?

Comment (wmertens) [inline src/immutable/formValues.js]: Wow! How do the tests even pass? 😕

Comment (wmertens) [inline src/immutable/formValues.js]: ah yes the tests create their own. Not ideal…

**Comment by wmertens** [comment]
Where do I go for writing docs?

**Comment by wmertens** [comment]
@erikras Alright, it's now testing the actually created `formValues`, and docs are added.

**Comment by erikras** [comment]
Published in [`v6.8.0`](https://github.com/erikras/redux-form/releases/tag/v6.8.0).

**Pull request merged**